### PR TITLE
Create EnchantmentCracker_zh_tw.properties

### PR DESCRIPTION
**Pull request by whats2000**

### CODE REVIEW

File: resources/i18n/EnchantmentCracker_zh_tw.properties [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+# Enchantment Calculator tab translations
+enchCalc.item.tooltip = 想要附魔的物品
+enchCalc.material.tooltip = 想要附魔的物品的材料
+enchCalc.maxBookshelves.tooltip = 使用中的有效書架數
+enchCalc.calculate = 開始計算
+enchCalc.calculate.tooltip = <html>按下來計算你<i>可能</i>需要丟出多少物品來獲得想要的附魔</html>
+enchCalc.playerSeedNotFound = 你需要先破譯玩家種子碼!\n先去第一個欄位裡看看吧
+enchCalc.impossible = 無匹配結果
+enchCalc.noDummy = 無虛設數值
```
Comment (Earthcomputer) [inline resources/i18n/EnchantmentCracker_zh_tw.properties:39]: Not an expert in traditional chinese, but I think this should be more along the lines of:
```suggestion
enchCalc.noDummy = 無额外個附魔
```

File: resources/i18n/EnchantmentCracker_zh_tw.properties [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+enchCalc.throwCount.tooltip = 需要丟出物品的數量
+enchCalc.slot.tooltip = 需要附魔的物品
+enchCalc.bookshelves.tooltip = 附魔所需的有效書架數量
+enchCalc.level = 等級:
+enchCalc.level.tooltip = 玩家當前的經驗等級。允許查找30級及以下的附魔
+
+# About tab translation
+# End each line with "\n\". Links must be the whole line, and are in the format "LINK <url> <text>".
+# When translating into other languages, make sure to indicate what links are in English, and instruct the user to report bugs in English.
+program.about = Enchantment Cracker %s\n\
```
Comment (Earthcomputer) [inline resources/i18n/EnchantmentCracker_zh_tw.properties:52]: This section needs translating as well

**Review by Earthcomputer — CHANGES_REQUESTED** [review]

**Comment by whats2000** [comment]
Sorry for sending too much comment at a same time,  the network have some issue. By the way, is this changes much better?

**Review by Earthcomputer — APPROVED** [review]
Looks good, thank you!

**Pull request merged**